docs/sdkx_ios/delegates.mdx

```diff
@@ -467,146 +441,145 @@ func handleHelpshiftEvent(_ eventName: String, withData data: [AnyHashable : Any
     ...
-    if([eventName isEqualToString:HelpshiftEventNameConversationRejected]) {
-        NSLog(@"Conversation rejected.");
+    if([eventName isEqualToString:HelpshiftEventNameSessionStarted]) {
+        NSLog(@"Helpshift session started.");
     }
 }
 ```
 
 </TabItem>
-<TabItem value="Swift Example" label="Swift Example">
+<TabItem value="Swift" label="Swift">
 
 ```swift
 func handleHelpshiftEvent(_ eventName: String, withData data: [AnyHashable : Any]?) {
-    if eventName == HelpshiftEventNameConversationRejected {
-        print("Conversation rejected")
-    }
+        if eventName == HelpshiftEventNameSessionStarted {
+            print("Helpshift session started.")
+        }
 }
 ```
 
 </TabItem>
 
 </Tabs>
 
-### Conversation Resolved Event {#conversationResolved}
+#### Helpshift Session Ended
 
-This event is triggered when an agent resolves the conversation.
+If you want to keep track of when helpshift session ends in your app, you can implement this delegate callback. The delegate will get fired every time the Helpshift session ends.
 
-Event name: `HelpshiftEventNameConversationResolved` <br />
-Event Data: `nil`
+- Event name: `HelpshiftEventNameSessionEnded` 
 
-Example:
-
-<Tabs>
+<Tabs groupId="ios-languages">
 
-<TabItem value="Objective-C Example" label="Objective-C Example">
+<TabItem value="Objective-C" label="Objective-C">
 
-```
+```objc
 - (void) handleHelpshiftEvent:(NSString *)eventName withData:(NSDictionary *)data {
     ...
-    if([eventName isEqualToString:HelpshiftEventNameConversationResolved]) {
-        NSLog(@"Conversation resolved.");
+    if([eventName isEqualToString:HelpshiftEventNameSessionEnded]) {
+        NSLog(@"Helpshift session ended.");
     }
 }
 ```
 
 </TabItem>
-<TabItem value="Swift Example" label="Swift Example">
+<TabItem value="Swift" label="Swift">
 
 ```swift
 func handleHelpshiftEvent(_ eventName: String, withData data: [AnyHashable : Any]?) {
-    if eventName == HelpshiftEventNameConversationResolved {
-        print("Conversation resolved")
-    }
+        if eventName == HelpshiftEventNameSessionEnded {
+            print("Helpshift session ended.")
+        }
 }
 ```
 
 </TabItem>
 
 </Tabs>
 
-### Conversation Reopened Event {#conversationReopened}
+### Unread Message Count Event {#unreadMessageCount}
 
-When resolution question is enabled, the users are asked if they're satisfied with the resolution. If the user rejects it and sends a new message, then the conversation is reopened and the Conversation Reopened event is triggered.
+If you want a count of all new messages received in an existing conversation, you can call this API `[Helpshift requestUnreadMesssageCount:];`.
 
-Event name: `HelpshiftEventNameConversationReopened` <br />
-Event Data: `nil`
+The count of the number of messages will be conveyed to your app via this event. You can also use this event to keep your badge counts updated.
```

Comment:
Can we change this to
The unread message count will be conveyed to your app via this event. You can also use this event to keep your badge counts updated.

OR 

The count of unread messages will be conveyed to your app via this event. You can also use this event to keep your badge counts updated.